test/test_kms_with_cli.py

```diff
@@ -62,6 +62,25 @@ def test_asymmetricKey_encrypt_decrypt(base_url, headers):
 
     print('====================test_asymmetricKey_encrypt_decrypt end===========================')
 
+def test_get_public_key(base_url, headers):
+    print('====================test_get_public_key start===========================')
+    key_SM2 = createkey.createkey(base_url, "EH_SM2", "EH_INTERNAL_KEY", None, None, None)
+    key_RSA_3072 = createkey.createkey(base_url, "EH_RSA_3072", "EH_INTERNAL_KEY", None, "EH_PAD_RSA_PKCS1_OAEP", None)
+    key_RSA_4096 = createkey.createkey(base_url, "EH_RSA_4096", "EH_INTERNAL_KEY", None, "EH_PAD_RSA_PKCS1_OAEP", None)
+    key_RSA_2048 = createkey.createkey(base_url, "EH_RSA_2048", "EH_INTERNAL_KEY", None, "EH_PAD_RSA_PKCS1", None)
+    key_EC_p224 = createkey.createkey(base_url, "EH_EC_P224", "EH_INTERNAL_KEY", None, None, "EH_SHA_2_224")
+    key_EC_p256 = createkey.createkey(base_url, "EH_EC_P256", "EH_INTERNAL_KEY", None, None, "EH_SHA_2_256")
+    key_EC_p384 = createkey.createkey(base_url, "EH_EC_P384", "EH_INTERNAL_KEY", None, None, "EH_SHA_2_384")
+    key_EC_p521 = createkey.createkey(base_url, "EH_EC_P521", "EH_INTERNAL_KEY", None, None, "EH_SHA_2_512")
+    asymmetricKey = [key_SM2, key_RSA_3072, key_RSA_4096, key_RSA_2048, key_EC_p224, key_EC_p256, key_EC_p384, key_EC_p521]
+
+    for i in asymmetricKey:
+        pubkey = getPubkey.getPublicKey(base_url, i)
+        print(i, ' public key:\n')
+        print(base64.b64decode(pubkey))
```

Comment:
refer to https://www.alibabacloud.com/help/en/key-management-service/latest/getpublickey
Can we return a decoded public key directly?